checks.d/go_expvar.py

```diff
@@ -54,6 +54,7 @@ class GoExpvar(AgentCheck):
 
     def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
+        self._namespace = init_config.get('namespace', DEFAULT_METRIC_NAMESPACE)
```

Comment:
I think this should be an instance level parameter instead. 
This way if you have multiple go apps running on the same host you can assign them a different namespace.
What do you think ?